simple/stats/cache.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2024 Google Inc.
```

Comment:
cache.py as the file-name for a specific cache seems a bit misleading, perhaps call it svg_cache or some such?

Reply:
I'd like to use this file for other cache data in the future as well. Will break it up if it gets too big.